src/cloud-api-adaptor/test/e2e/common_suite.go

```diff
@@ -35,6 +35,18 @@ func DoTestCreateSimplePod(t *testing.T, e env.Environment, assert CloudAssert)
 	}
 }
 
+func DoTestLibvirtCreateSimplePodWithSecureCommsIsValid(t *testing.T, e env.Environment, assert CloudAssert) {
+	if !testCase_secureComms_isActive {
+		t.Skip("Skip - SecureComms is configured to be inactive - no need to test")
+	}
+	pod := NewBusyboxPodWithName(E2eNamespace, "simple-test-with-security-comms-is-active").GetPodOrFatal(t)
+	if isTestOnCrio() {
+		NewTestCase(t, e, "SimplePeerPodWithSecureComms", assert, "PodVM is created with secure comms").WithPod(pod).WithSecureCommsIsActive().Run()
+	} else {
+		NewTestCase(t, e, "SimplePeerPodWithSecureComms", assert, "PodVM is created with secure comms").WithPod(pod).WithNydusSnapshotter().WithSecureCommsIsActive().Run()
+	}
```

Comment:
I don't see the need to test that nydus is used here, so can you just run a single test case for containerd and crio please.